screens/DaterProfile.tsx

```
@@ -0,0 +1,239 @@
+            ) : (
+              <ThemedText>No friends added yet</ThemedText>
+            )}
+          </View>
+        )}
+      </View>
+    </ScrollView>
+  );
+};
+
+const styles = StyleSheet.create({
+  container: {
```

ESLint

[error] Expected indentation of 4 spaces but found 2.

(@stylistic/js/indent)